README.md

```diff
@@ -1,24 +1,80 @@
-# Rust binding to [Open Quantum Safe][oqs]'s [liboqs][]
+liboqs-rust: Rust bindings for liboqs
+=====================================
 
-This crate provides bindings to [liboqs][]. It provides unsafe `ffi` bindings in the `oqs-sys` crate.
-Safe wrappers are offered via the `oqs` crate.
+[![Build status](https://github.com/dstebila/oqs-rs/workflows/Continuous%20integration/badge.svg)](https://github.com/dstebila/oqs-rs/actions?query=workflow%3A"Continuous+integration")
 
+**liboqs-rust** offers a Rust wrapper for the [Open Quantum Safe](https://openquantumsafe.org/) [liboqs](https://github.com/open-quantum-safe/liboqs/) C library, which is a C library for quantum-resistant cryptographic algorithms.
 
-## Adding new algorithms
+Pre-requisites
+--------------
 
-### KEMs:
+liboqs-rust depends on the [liboqs](https://github.com/open-quantum-safe/liboqs) C library; liboqs must first be compiled as a Linux/macOS/Windows library (i.e. using `ninja install` with `-DBUILD_SHARED_LIBS=ON` during configuration), see the specific platform building instructions below.
+
+<span style="color: red;">TODO: Check?</span>
+
+Contents
+--------
+
+This crate provides unsafe `ffi` bindings in the `oqs-sys` crate, and safe wrappers are offered via the `oqs` crate.
+
+Usage
+-----
+
+<span style="color: red;">TODO</span>
+
+Running
+-------
+
+<span style="color: red;">TODO</span>
```

Comment:
This should also links to the crate documentation, for as far as this is even relevant (how do you 'run' a library)

Reply:
If there was an example program, I guess this is where it would go.  If there's no example program, then we can remove this.

Reply:
I haven't written any example programs. The tests/doctests cover the most basic examples anyway